hack/update-deps.sh

```diff
@@ -43,6 +45,10 @@ function delete_contour_cluster_role_bindings() {
   sed -e '/apiVersion: rbac.authorization.k8s.io/{' -e ':a' -e '${' -e 'p' -e 'd'  -e '}' -e 'N' -e '/---/!ba' -e '/kind: ClusterRoleBinding/d' -e '}'
 }
 
+function rename_cluster_role() {
+  sed -e "/apiVersion: rbac.authorization.k8s.io/{N;/kind: ClusterRole\b/{N;N;N;s/name: contour/name: $1/}}"
```

Comment:
I think after this PR we should replace sed with `ytt` - these expressions are now beyond me lol

Reply:
`ytt` would be awesome! 😬